lib/src/util/react_wrappers.dart

```diff
@@ -62,7 +62,7 @@ dynamic getInstanceKey(ReactElement instance) => instance.key;
 dynamic getInstanceRef(ReactElement instance) => instance.ref;
 
 /// Returns whether an [instance] is a native Dart component (react-dart [ReactElement] or [ReactComponent]).
-bool isDartComponent(/* [1] */ instance) {
+bool isDartComponent(/* ReactElement|ReactComponent */ instance) {
```

Comment:
This is actually `ReactElement|ReactComponent|Element`, since we support and anticipate DOM component instances being passed in here.